tests/python/relay/test_ir_nodes.py

```diff
@@ -79,6 +105,14 @@ def test_type_relation():
     assert tr.args == args
     assert tr.num_inputs == num_inputs
 
+    back = json_roundtrip(tr)
+    # assert back == tr
+    assert tr.num_inputs == back.num_inputs
+    assert len(back.args) == len(tr.args)
+    for i in range(len(back.args)):
+        assert back.args[i] == tr.args[i]
+    assert back.attrs.name == tr.attrs.name
```

Comment:
Now that there's an AttrVisitor, should I change alpha-equality to use it for checking deep equality of attrs?